packages/react-components/src/components/xy-output-component.tsx

```diff
@@ -159,6 +163,17 @@ export class XYOutputComponent extends AbstractTreeOutputComponent<AbstractOutpu
             events: [ 'mousedown' ],
         };
         // width={this.props.style.chartWidth}
+        if (this.state.outputStatus === ResponseStatus.COMPLETED && this.state.xyTree.length === 0 ) {
```

Comment:
> @IbrahimFradj: I noticed that the message does not display with **Event Matching Latency - Latency vs. Time** and with **Memory Latency vs. Time**.
> I also noticed that for the **Memory Usage** view, sometimes it displays with some traces and sometimes it doesn't display and gets stuck in Analysis running. (ex: with the **apt** trace it works but with **ls-ust** it doesn't)

My understanding is this message only displays when then analysis returns status "COMPLETE", but with an empty model. 

Could you check whether you're observing cases where the analysis returns "FAILED"? The case where the analysis returns "FAILED" is still not managed #460.

Reply:
Yes I can look at that

Reply:
To clarify, @bhufmann said earlier that this PR doesn't need to manage the FAILED case. If the problems @IbrahimFradj noticed are with the FAILED case then we can still merge this PR.

![image](https://user-images.githubusercontent.com/28311615/134514829-c6842cbd-9214-42d5-a59d-dc39e97380cd.png)
Source: [Comment thread](https://github.com/theia-ide/theia-trace-extension/pull/447#discussion_r695193161)